### ISSUE TITLE
Additional Payment Options

### ISSUE THREAD

**Issue by ZenRevision** [gileli121/WindowTop#380]
It would be great if you could add the _Pay with Amazon_ payment option. I'm not sure if there are any other Amazon payment providers, but [fastspring.com](https://fastspring.com) is providing this payment service which allows online vendors to receive payments through Amazon. It looks like it should be fairly simple to integrate.

I've used it a few times now, purchasing other products/services online -- it works great, and means I don't need to input my credit card details for another payment service which I always try to avoid having to do anyway. This is even more important to me right now though, because I'm traveling abroad, and won't be back in the US for the next couple years. My current location creates additional security concerns for me, such that I don't think I'll be sending any banking/credit card details over the network through my local systems here for at least a few months.

An example of a site using FastSpring for Amazon payments: [Stardock.com](https://www.stardock.com) Stardock Systems is a software company that has been in business for over 25 years, I think.

Whenever outside the USA, I have had endless problems with PayPal due to their anti-VPN/proxy and geofencing policies.

### However, I really want to pay you for your _excellent_ app!

If not the _Pay with Amazon_ option mentioned above, I think Apple Pay could also work. Otherwise perhaps we could work out a way for me to send payment to you via Zelle (which many US banks bundle with their accounts, but I think people can create Zelle accounts even if their bank doesn't bundle it by default).

**Comment by gileli121** [comment]
Interesting idea.
Thank you.
I was unaware of the new payment methods that came out recently.
It is not something that I will have time to work on during this weekend.

If PayPal isn't an option for you, the Microsoft Store offers a flexible alternative for your purchases.
The payment is handled 100% by their system, and they support not only PayPal.

Please note that if you purchase from their store, you buy a different activation method implemented by the Microsoft Store ecosystem. You will not get any license key. You can use this activation by logging in to their store and installing the app. Such activation is supported only on the Microsoft Store version.

**Comment by ZenRevision** [comment]
Yeah, same problem with MS Store. If they already had my payment info on file... but they don't. So I would have to send MS my credit card info from my current location, which I must avoid doing for the time being. I'm not a fan of MS anyway, and I definitely don't want MS connecting my local (offline) windows account to my online MS live services account.

Amazon, Apple Pay, and Zelle, already have my payment info on file, so I wouldn't need to send my bank details/or card number from my location.

More payment options would also allow more people, to more easily pay you for your app anyway though. I know I'm not the only one who has run into problems with PayPal. Lots of people already have Amazon and Apple Pay accounts set up, and are comfortable with their security.

Zelle is more of a personal (as opposed to business) service for sending money to contacts, but I've used it before and it's reliable. Probably not suitable as a payment option on your website though.

If you have an Etsy sellers account... I've never bought anything on Etsy, but I have a sellers account, so they have my payment details, so maybe it's possible I could get money to you that way. Seems like that may be slightly complicated, but potentially workable.

Otherwise, I guess I can see if one of my Chinese contacts here is familiar with your Chinese payment options and willing to facilitate the transaction. I'm in China for the next few months, but I'm not Chinese, and don't have any Chinese bank accounts - but I'll see what I can do.

**Comment by ZenRevision** [comment]
Okay, I got a friend to pay for it through lizhi.shop with their alipay. Thanks!

Still think you should add the _Amazon_ payment option via _FastSpring_ though... and _Apple Pay._ Because **you'd likely get more sales.**

**Comment by gileli121** [comment]
@ZenRevision 
Thanks!
I will probably do it. It's just that I don't have time to do it now
